system-contracts/test/L1Messenger.spec.ts

```diff
@@ -56,7 +63,7 @@ describe("L1Messenger tests", () => {
     await l1Messenger
       .connect(bootloaderAccount)
       .publishPubdataAndClearState(
-        ethers.constants.AddressZero,
+        L2_DA_COMMITMENT_SCHEME.EMPTY,
```

Comment:
I thought that AddressZero = None, while Empty is for Validium, which still sends the data hash to L1? maybe

Reply:
also below in this file

Reply:
I believe it doesn't really matter here. NONE is invalid value, so call will be reverted